examples/server-demo/src/server.js

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+
+import App from './App';
+import html from './htmlTemplate';
+
+const PORT = process.env.PORT || 5657;
+const app = express();
+
+app.get('/', (req, res) => {
+  const app = ReactDOMServer.renderToString(<App />);
+
+  const initResults = {
+    recommendations: [
+      {
+        name: 'Sneakers Hogan Rebel pink',
+        objectID: 'MX1234',
+        price: {
+          value: 278,
+          currency: 'EUR',
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
You could use the instruction at the file level only once, at the top:

```js
/* eslint-disable @typescript-eslint/camelcase */
```

Reply:
✅